components/automate-cli/pkg/verifyserver/services/externalpostgresqlservice/externalpostgresqlservice.go

```diff
@@ -0,0 +1,121 @@
+package externalpostgresqlservice
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/constants"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/utils/db"
+	"github.com/chef/automate/lib/logger"
+	_ "github.com/lib/pq"
+)
+
+type ISExternalPostgresqlService interface {
+	GetPgConnection(*models.ExternalPgRequest) (*models.ExternalPgResponse, error)
+}
+
+type ExternalPostgresqlService struct {
+	logger  logger.Logger
+	DBUtils db.DB
+	Req     *models.ExternalPgRequest
+}
+
+func NewExternalPostgresqlService(db db.DB, logger logger.Logger) ISExternalPostgresqlService {
```

Comment:
NewExternalPostgresqlService should return an object of type ExternalPostgresqlService but not the interface

Reply:
done